ocean_lib/ocean/ocean.py

```diff
@@ -102,6 +103,11 @@ def __init__(
     def OCEAN_address(self) -> str:
         return get_ocean_token_address(self.config.address_file, web3=self.web3)
 
+    @property
+    @enforce_types
+    def OCEAN_token(self) -> ERC20Token:
```

Comment:
I like this utility and I think there are many places where we do get_token(... ocean.OCEAN_address) instead. Can you refactor those to use OCEAN_token itself? I would also suggest the naming OCEAN_datatoken instead, for conformity.

Reply:
Sure I can refactor places where we use `ERC20Token(web3, ocean.OCEAN_address)` to use this `ocean.OCEAN_token` method instead.

I disagree with renaming to `OCEAN_datatoken` because the underlying OCEAN token contract does not support all the datatoken methods, so calling it a "datatoken" seems misleading to me.

Honestly, now that I'm thinking about it, the fact that the class `ERC20Token` isn't instead called `Datatoken` like it used to be, is also misleading.

Reply:
I refactored all the uses of `ERC20Token(web3, get_address_of_type(config, "Ocean"))` to instead use `ocean.OCEAN_token` or a new `OCEAN_token` fixture I added.

I did not rename `ERC20Token`, `ERC721NFT`, or `BTokenBase`.  I'll leave these possible refactors for a future issue.

Reply:
created https://github.com/oceanprotocol/ocean.py/issues/832 and resolving comment.